kubernetes/elastic-helm/daemonset.yaml

```diff
@@ -14,7 +14,7 @@ image:
   tag: 8.15.0-SNAPSHOT
 command:
   name: "/usr/share/elastic-agent/elastic-agent"
-  extraArgs: ["otel", "-c", "/etc/elastic-agent/otel.yaml"]
+  extraArgs: ["otel"]
```

Comment:
🤔 does this mean that a default config file will be used? Actually is `/etc/elastic-agent/otel.yaml` the default?

Reply:
The default file is not used, as the Helm Chart dynamically creates a file from the [provided config map](https://github.com/elastic/opentelemetry-demo/blob/main/kubernetes/elastic-helm/daemonset.yaml#L47) and populate the `--config/-c` parameter afterward.

See https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-collector/templates/_pod.tpl#L24

The actual image arguments in the manifest generated by Helm look like: `-c /etc/elastic-agent/otel.yaml --config /conf/relay.yaml`

Reply:
Right, got it! Thank's for clarifying!